packages/cli-lib/lib/watch.js

```diff
@@ -16,11 +17,35 @@ const { upload, deleteFile } = require('../api/fileMapper');
 const escapeRegExp = require('./escapeRegExp');
 const { convertToUnixPath, isAllowedExtension } = require('../path');
 const { triggerNotify } = require('./notify');
+const { getThemeJSONPath } = require('./files');
+const { getEnv } = require('./config');
+const { getHubSpotWebsiteOrigin } = require('./urls');
+const { ENVIRONMENTS } = require('./constants');
 
 const queue = new PQueue({
   concurrency: 10,
 });
 
+const _notifyOfThemePreview = (filePath, accountId) => {
+  if (queue.size > 0) return;
+
+  const themeJSONPath = getThemeJSONPath(filePath);
+  if (!themeJSONPath) return;
+  const pathParts = themeJSONPath.split('/');
+  if (pathParts.length < 2) return;
+  const themeFolder = pathParts[pathParts.length - 2];
+
+  const baseUrl = getHubSpotWebsiteOrigin(
+    getEnv() === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
+  );
+
+  logger.log(`
+  To preview this theme, visit:
+  ${baseUrl}/theme-previewer/${accountId}/edit/${themeFolder}
```

Comment:
Can we utilize some of the utils in links.js for this url?

Reply:
I dont think so, none of those have a concept of a dynamic URL.  They only insert portal id and base url